src/planscape/planscape/urls_v2.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, include
+from projects.routers import router as projects_router
+from metrics.routers import router as metrics_router
+from goals.routers import router as goals_router
+from planning.routers import router as planning_router
+
+app_name = "api"
```

Comment:
I've been wondering if there's a benefit to having a version here, instead of just "api" -- e.g., "api2" or "v2" -- to be clear why some endpoints might belong under this prefix, and some don't.
I don't think it's necessary, but I've got mixed feelings about it.